include/yaml-cpp/node/node.h

```diff
@@ -128,7 +128,7 @@ class YAML_CPP_API Node {
   void AssignData(const Node& rhs);
   void AssignNode(const Node& rhs);
 
- private:
+ protected:
```

Comment:
Rather than making these attributes `protected` (and those below in `nodebuilder.h`), it might be a better idea to add `protected` getters (which of course I'm willing to do, if the general thrust of this PR is reasonable to you).